app/components/sidebars/settings/settings_sidebar_base.js

```diff
@@ -129,7 +133,7 @@ export default class SettingsSidebarBase extends PureComponent {
         logout();
     });
 
-    openModal = async (screen, title, passProps = {}) => {
+    openModal = async (screen, title, passProps = {}, alignTitleCenter = false) => {
```

Comment:
Although I see that in Figma the title is centered, the Figma file is also showing iOS, on Android the title is aligned to the left, so this change here is not required and it should be removed.

Reply:
Done.